cxx-qt-lib/include/qt_types.h

```diff
@@ -27,6 +27,30 @@
 namespace rust {
 namespace cxxqtlib1 {
 
+template<typename R, typename T>
+struct cxx_qt_convert
+{
+  R operator()(T val) { return val; }
+};
+
+template<typename R, typename T>
+struct cxx_qt_convert<R&, T>
+{
+  R operator()(T val) { return val; }
```

Comment:
This should return `R&`, shouldn't it?
Also, this would need to take a `T& val`